src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -320,33 +270,23 @@ private void Execute(ContractEntry entry, StateUpdate stateUpdate, ITxTracer txT
             }
             catch (Exception ex) when (ex is EvmException || ex is OverflowException)
             {
-                if (_logger.IsEnabled(LogEventLevel.Verbose))
-                {
-                    _logger.Verbose($"EVM EXCEPTION: {ex.GetType().Name}");
-                }
-                
+                if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose($"EVM EXCEPTION: {ex.GetType().Name}");
+
                 _stateProvider.Restore(stateSnapshot);
                 _storageProvider.Restore(storageSnapshot);
             }
 
-            if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
-                _logger.Verbose("Gas spent: " + spentGas);
-            }
+            if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose("Gas spent: " + spentGas);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)